app_handler.go

```diff
@@ -37,11 +37,11 @@ func (app *App) handler() func(*fasthttp.RequestCtx) {
 		}
 		app.middlewaresMu.RUnlock()
 		if len(app.domains) > 0 {
-			d := string(ctx.URI().Host())
+			d := ctx.URI().Host()
```

Comment:
Please revert this change. You forgot to move string()

Reply:
String used in map indexing: https://github.com/toby3d/gramework/blob/da22343551eae224e89a5505277c0d1f25cdb900/app_handler.go#L42-L44

Reply:
Yes, and one call is better then two calls, so we should just declare `d` as a string

Reply:
```
app_handler.go:40:15:warning: m[string(key)] would be more efficient than k := string(key); m[k] (SA6001) (megacheck)
```